lib/doofinder_api_items.php

```diff
@@ -19,6 +19,11 @@
 
 class DoofinderApiItems
 {
+    private $hashid;
+    private $api_key;
+    private $api_url;
+    private $type;
+
```

Comment:
Delete this. In this file you do not have to make changes

Reply:
I know that this file is unrelated, but I applied the boy-scout rule and I tried to fix them, but I'm going to revert them if you consider it's better to apply them on a different PR :smiley: